### PR TITLE
Foundations/Introduction to Flexbox: Add instructions for how to uncomment CSS code

### PR DESCRIPTION
**Pull request by takinabradley**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**
I've noticed a couple instances where learners have mistakenly deleted code comments in the codepen, instead of uncommenting them. I think it would be a good idea to instruct learners how code comments work, and how to 're-enable' the lines.

**2. This PR:**
- Adds instructions for how to uncomment CSS code

**3. Additional Information:**
Start of a discord discussion, where a learner had this exact issue: 
https://discord.com/channels/505093832157691914/505093832157691916/962089437792186409

### CODE REVIEW

File: foundations/html_css/flexbox/flexbox-intro.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Flexbox is a way to arrange items into rows or columns. These items will flex (i
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-We'll get into exactly what's going on here soon enough. But for now, uncomment the two flex related lines in the above CSS and check out the result. All 3 divs should now be arranged horizontally. If you resize the results frame with the "1x", ".5x" and ".25x" buttons you'll also see that the divs will 'flex'. They will fill the available area and will each have equal width.
+We'll get into exactly what's going on here soon enough. But for now, uncomment the two flex related lines in the above CSS and check out the result. CSS comments prevent the browser from interpreting lines as code, and are wrapped between `/*` and `*/` tags. The lines can be 're-enabled' by removing the comment tags surrounding unused code.
```
Comment (thatblindgeye) [inline foundations/html_css/flexbox/flexbox-intro.md:30]: Like the addition of mentioning comments within code! Wondering if this paragraph could be worded a bit differently for a better flow. What do you think of something along the lines of:

```suggestion
We'll get into exactly what's going on here soon enough. But for now, let's uncomment the two flex related CSS declarations in the above Codepen by removing the `/*` and `*/` tags surrounding them, then check out the result.

**A Note on Comments**: Comments prevent the browser from interpreting lines as code, and are wrapped between specific tags. CSS uses `/*`as an opening comment tag and `*/` as a closing comment tag, while HTML and JavaScript have their own syntax. Commented out lines of code can be 're-enabled' simply by removing the comment tags surrounding the code. 
```

Comment (takinabradley) [reply foundations/html_css/flexbox/flexbox-intro.md:30]: I had originally intended on adding a note similar to this, but I felt adding a note about comments closely preceding the note about the codepen created a slight distraction in the flow of the lesson. 

After looking over it again with the preview tool, the way you inserted the note seems a lot less distracting than the way I would have. I like it!

Comment (thatblindgeye) [reply foundations/html_css/flexbox/flexbox-intro.md:30]: Hey @takinabradley , just wanted to check if you were still looking to make this update?

Comment (takinabradley) [reply foundations/html_css/flexbox/flexbox-intro.md]: Oh, sorry! I didn't realize I had to push your suggested changes, and was waiting to hear back from you. It's done now!

**Review by thatblindgeye — APPROVED** [review]
Thank you for letting me know of the confusion, and also for making this update! 🙂

**Pull request merged**